### PR TITLE
SAK-30433 insertSpinnerAfter was using override of an element that did not exist

### PR DESCRIPTION
**Pull request by ottenhoff**
Blocker issue because grades can no longer be released.

/CC: @bjones86 does this look okay?

### CODE REVIEW

File: reference/library/src/webapp/js/spinner.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ SPNR.insertSpinnerAfter = function( clickedElement, escapeList, overrideSpinnerL
     spinner.className = "spinPlaceholder";
 
     // If an override location for the spinner is desired, append it as a child...
-    if( overrideSpinnerLocation !== null && overrideSpinnerLocation !== "" )
+    if( overrideSpinnerLocation !== null && overrideSpinnerLocation !== "" && document.getElementById( overrideSpinnerLocation ) )
```
Comment (ern) [inline reference/library/src/webapp/js/spinner.js]: Should it be sufficient to say:
if( overrideSpinnerLocation && document.getElementById( overrideSpinnerLocation )

**Comment by bjones86** [comment]
Looks good, I should have removed this when I refactored it originally, thanks @ottenhoff.

Also, I think @ern's suggestion is correct; easier to read that way too.

**Comment by ottenhoff** [comment]
Thanks for review. I just implemented the @ern suggestion and squashed/rebased

**Comment by bjones86** [comment]
Great, thanks @ottenhoff!

**Pull request merged**